EventListener/SerializerListener.php

```diff
@@ -5,18 +5,18 @@
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\Serializer\Exception\ExceptionInterface as BaseSerializerException;
-use Symfony\Component\Serializer\SerializerInterface;
+use Xabbuh\XApi\Serializer\SerializerRegistryInterface;
 
 /**
  * @author Christian Flothmann <christian.flothmann@xabbuh.de>
  */
 class SerializerListener
 {
-    private $serializer;
+    private $serializerRegistry;
 
-    public function __construct(SerializerInterface $serializer)
+    public function __construct(SerializerRegistryInterface $serializerRegistry)
```

Comment:
When we add support for more API endpoints, we can add arguments for the other serializer types. But by injecting them directly we will not violate the law of Demeter and stay clear about what exactly we intend to use (right now it's not clear that we just make use of the statement serializer).